index.html

```diff
@@ -15,6 +15,20 @@
     </style>
 </head>
 <body>
-  <h1><center>No.</center></h1>
+  <h1><center id="injury-status">Yes.</center></h1>
```

Comment:
are we sure we want to assume Ben is hurt? What if the default text is Loading... or empty string?